TypeInjections/Tool/README.md

```diff
@@ -0,0 +1,8 @@
+## typeCart features
+typeCart offers multiple ways to compare two projects
+- projects on local computer:
+    * `$ typecart -p [OldFolderPath] [NewFolderPath] [OutputPath]`
+- git commits on a remote repository:
+    * `$ typecart -r https://gitub.com/[org]/[repo].git [git commit SHA1] [git commit SHA1] [OutputPath]`
```

Comment:
I'd say `old git commit SHA1` and `new git commit SHA1` here (also below).

Does one need to provide the full SHA1 hash or is a short version or something like `HEAD` acceptable here?

Maybe you can give some examples in the README.